### ISSUE TITLE
Add entry to FAQ: Recommended procedure when switching to a new phone

### ISSUE THREAD

**Issue by MikeJayDee** [corona-warn-app/cwa-website#411]
I will shortly be switching to a new phone so I was wondering what the correct procedure in terms of the CWA is. I expected a FAQ entry on this, but there is just a mention of an iOS bug if one uses iTunes to migrate the app with the recommendation to simply install the new one.

In the first place, there should definitely be a FAQ entry on this. As to the content of the FAQ entry:

- First, the new FAQ entry should mention that the data on collected and sent keys can currently not be transferred to a new phone on both Android and iOS, and that the CWA should simply be newly installed on the new phone as soon as possible in all cases.

- Secondly, one could give the advice to those that wanted to maximise continuity to carry both phones with them for a period of two weeks.  In case of a positive test result within those two weeks, the old phone should be used to scan the QR code and submit the diagnosis keys.

- Alternatively, one could advise that the old phone could be left at home and only be turned on once a day for two weeks to check for risk encounters. This would be sufficient to receive warnings about risk encounters from days where only the old phone was carried; however, in the unlikely event of receiving a positive test result oneself, one would require an extra TeleTAN to be able to submit diagnosis keys from both phones (I'm not sure whether this is supposed to be possible in such a case).

Note that there are some related issues already (without proper resolution), but those are not concrete requests for a FAQ entry, so I opened an issue her:

- https://github.com/corona-warn-app/cwa-wishlist/issues/37

- https://github.com/corona-warn-app/cwa-documentation/issues/82

**Comment by NKarolak** [comment]
> I expected a FAQ entry on this

The FAQs are actually saying that you cannot restore CWA data at all.
https://www.coronawarn.app/en/faq/#backup

**Comment by Ein-Tim** [comment]
A suggestion:

**German**: 

>Um sicherzugehen das Sie über alle Kontakte informiert werden, ist es notwendig, dass sie auf ihrem alten Gerät die Corona-Warn App noch 14 Tage lang regelmäßig öffnen, um zu sehen, ob diese Ihnen Begegnungen anzeigt.
Sollten Sie innerhalb dieser 14 Tage positiv getestet werden, sollten Sie das Testergebnis auf beiden Geräten teilen, damit alle Kontakte informiert werden können. 
Eine Anleitung, wie sie Ihr Testergebnis auf mehreren Geräten teilen könnn, finden Sie [hier](https://www.coronawarn.app/de/faq/#test_multiple_devices).

**English**:

>To make sure that you are informed about all contacts, it is necessary that you open up the Corona-Warn App on your old device regularly for 14 days to see if it shows encounters.
If you test positive within these 14 days, you should share the test result on both devices so that all contacts can be informed. For instructions on how to share your test result on multiple devices, see [here](https://www.coronawarn.app/de/faq/#test_multiple_devices).

**Comment by MikeMcC399** [comment]
@Ein-Tim 
Thank you for raising this topic again. It could be more relevant over Christmas if people get presents of a new mobile phone and want to get uninterrupted use of CWA.

The first point is that it needs the basic willingness of the Open Source Team to support publication of instructions. I would like to see some feedback from them about what they think about that.

Secondly, I think it should be a separate FAQ article that can have its own link, like https://www.coronawarn.app/en/faq/#new_device_migration and not have to share a link with the backup topic https://www.coronawarn.app/en/faq/#backup.

Thirdly, we need to agree on what the user should do. @MikeJayDee made a good start in the original posting.

I would start off assuming that the old phone is no longer needed for any other purposes once the new phone is active.

- Explain that it is not possible to transfer data from one phone to another, therefore the phones have to run in parallel for 14 days.

On the old phone:
- Make sure that the Corona-Warn-App has EXPOSURE LOGGING ACTIVE showing as its status.
This means that regular exposure checks can take place against newly submitted Diagnosis Keys from other users.
- Disable Bluetooth.
This stops any new Rolling Proximity Identifiers (RPI) being exchanged with other mobile devices.
- Leave the phone at home, connected to a charger, with WiFi or mobile network connectivity and check for notifications each day.
Ensures that exposure checks are done every 6 hours (WiFi) or once a day (mobile network only), keeps the phone generally available and makes sure that the user knows if there is any notification of increased risk requiring action.
- After a full 14 days have elapsed from the time that Corona-Warn-App was installed on the new phone, uninstall the Corona-Warn-App from the old phone.
After 14 days none of the stored keys are relevant for risk calculation, there are no new keys and therefore the device and the app are no longer needed for the purposes of the Corona-Warn-App.

On the new phone:
- Install Corona-Warn-App and enable exposure logging.

For testing:
- If the user is tested during the period when their old and new phones are active, enter any QR code on the new phone. If the test result is positive, share that on the new phone. In addition request a TAN and share this on the old phone.

If we can agree that something like this should be published, and what the best recommendations for users are, then we can put this into suitable text and get it included in the FAQs. I'm not sure whether to just give users a "cook-book" explanation of what they should do or whether to also include the information about why each step is recommended. To keep it simple though, I would tend towards the cook-book approach.

**Comment by dsarkar** [comment]
@Ein-Tim @MikeMcC399 

Looks good so far, and this might just do it.  One small point though, using both phones for 14 days: 

- This solves the problem that all your contact will be informed if you are tested positive.
- But my understanding is that the assessment of your own risk is slightly affected since you stop accumulating certain encounters with a higher risk score.


However, it seems the mosty reasonable solution.

Best wishes,
DS

---
Corona-Warn-App Open Source Team

**Comment by MikeMcC399** [comment]
@dsarkar 
> ... it seems the most reasonable solution.

Thanks for your feedback! I too think that it is the best compromise given that there is no technical path to transfer data from an old phone to a new phone.

Let me know if you want a text suggestion for an FAQ article and I will submit a PR. If the Open Source Team prefer to write it yourselves, that is also fine.

**Comment by heinezen** [comment]
@MikeMcC399 I have already started a draft based on your comment. However, I could use your input to review the PR once it is ready.

**Comment by MikeMcC399** [comment]
@heinezen 
> I have already started a draft based on your comment. However, I could use your input to review the PR once it is ready.

I'll be happy to collaborate with you on this. 🙂

**Comment by heinezen** [comment]
@MikeMcC399 See https://github.com/corona-warn-app/cwa-website/pull/653

**Comment by jensb** [comment]
I'm not sure if this is the same issue, but how about transferring / saving / exporting 
- certificates (tests, vaccinations, etc)
- manually saved locations and encounters (diary, check-in)
? If you move to a new phone, at least *this* data should be migrated.

**Comment by MikeMcC399** [comment]
@jensb 

>  ... how about transferring / saving / exporting
>
> - certificates (tests, vaccinations, etc)
> - manually saved locations and encounters (diary, check-in)
>
> ? If you move to a new phone, at least this data should be migrated.
 
Are you asking for the app to do this or are you requesting that the FAQ should be extended to describe how to do what you request?

**Comment by jensb** [comment]
Actually, I would expect the Google One backup (or migration utilities like Samsung Switch) to do this for me since it *supposedly* migrates all apps and app data. I do not want to create yet another cloud account somewhere just to be able to sync data between smartphones.
Maybe the way CWA saves this data needs to be changed?

**Comment by MikeMcC399** [comment]
@jensb

Thank you for explaining your expectations.

There is an open issue from @dsarkar 
- https://github.com/corona-warn-app/cwa-website/issues/1319

talking about extending the FAQs to cover the backup topic more extensively. It has been open for a long time though without any progress. You might like to add your comments there since the issue here, which you have posted to, is already closed.

There is also a closed issue from @Ein-Tim 
- https://github.com/corona-warn-app/cwa-website/issues/1766

discussing about this topic, which might be helpful for background reading.

You could request a change to the app by opening an [issue in the Android repository](https://github.com/corona-warn-app/cwa-app-android/issues), however I don't think it is likely that backup will be enabled on Android. (It is explicitly disabled in the [app manifest](https://github.com/corona-warn-app/cwa-app-android/blob/main/Corona-Warn-App/src/main/AndroidManifest.xml): `android:allowBackup="false"`).  The reasons given in the past concerned the protection of personal health data.
